test/websocket-server.test.js

```diff
@@ -724,4 +724,24 @@ describe('WebSocketServer', function () {
       });
     });
   });
+
+  describe('Server address', function () {
```

Comment:
Please move this block after `describe('#ctor', ...);` and use `'#address'` as title